CodedEvents/backend/models.py

```diff
@@ -1,3 +1,56 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+
+class Role(models.Model):
+    name = models.CharField(max_length=30)
+    desc = models.CharField(max_length=100)
+
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    roles = models.ManyToManyField(Role)
+
+
+class Address(models.Model):
+    street = models.TextField
```

Comment:
Why choose TextField instead of Char? Any particular reason here?

Reply:
That's a good point, CharField may make more sense since these entries shouldn't be that long